### PR TITLE
Disable geometry buttons instead of hiding

### LINKED ISSUE #11543: Disable geometry edition button in read-only layers
After a small discussion in #10905, @saleiva suggested to show always the geometry edition buttons but disabling them in case of read-only layers. Comment link: https://github.com/CartoDB/cartodb/issues/10905#issuecomment-278362912

image

Current result is that we hide that button.

image

A read only layer can be obtained doing an analysis on that layer.

Not sure if we have a defined style for disabling those kind of button. Summoning here @CartoDB/design

### PR DESCRIPTION
**Pull request by rubenmoya**
Fixes #11543

test

### CODE REVIEW

File: lib/assets/core/test/spec/cartodb3/editor/layers/layer-content-view.spec.js [hunk 1]
```diff
@@ -147,8 +143,23 @@ describe('editor/layers/layer-content-view/lcv', function () {
 
   describe('._renderContextButtons', function () {
     var analysisDefNodeModel;
+    var layerDefinitionModel;
+    var queryGeometryModel;
+    function hasTipsy (subviews) {
```
Comment (nobuti) [inline lib/assets/core/test/spec/cartodb3/editor/layers/layer-content-view.spec.js]: nice!

**Comment by rubenmoya** [comment]
It makes sense, to move it to the container, we thought about it but decided to put it in the items just in case we wanted to change the text later, but you're right.

**Comment by nobuti** [comment]
Works like a charm ✨

**Comment by nobuti** [comment]
Congrats @rubenmoya @rjimenezda, your first PR is about to be merged 🎉

**Comment by xavijam** [comment]
VAMOS!

**Pull request merged**